/**:
  ros__parameters:
    L0: 2.3
    dt: 0.05
    # Trailer Configuration (List of dictionaries)
    # Since ROS 2 params don't support list of dicts easily, we use parallel arrays or flattened structure.
    # For simplicity in this example, we'll use parallel arrays.
    trailer_L_bars: [1.0, 1.0, 1.0, 1.0]
    trailer_L_trls: [1.2, 1.2, 1.2, 1.2]
    
    # Hitch Calculation
    # Hitch Offset = Overhang + Tail Length
    tractor_tail_length: 0.15 # Distance from Tractor Rear Face to Hitch 1
    trailer_tail_length: 0.15 # Distance from Trailer Rear Face to Hitch (for all trailers)
    
    # Visualization Colors [r, g, b, a]
    color_tractor: [1.0, 0.27, 0.0, 0.5] # OrangeRed
    color_trailer: [0.0, 0.0, 1.0, 0.5]  # Blue
    color_wheel: [1.0, 1.0, 1.0, 1.0]    # White
    color_drawbar: [0.8, 0.9, 1.0, 1.0]  # White Blue (Light Blue)

    # Visualization Dimensions [m]
    tractor_width: 1.8
    tractor_front_overhang: 0.1 # Distance from Front Axle to Front Face
    tractor_overhang: 0.4 # Distance from Rear Axle to Rear Face (Rear Overhang)
    trailer_width: 1.8
    trailer_body_len: 2.0
    trailer_overhang: 0.4
    wheel_diam: 0.6
    wheel_width: 0.3
    track_width: 1.6 # Distance between left and right wheels
